tests/WebUI/Utils.test.ts

```diff
@@ -113,4 +113,37 @@ describe("Utils getPillTags value check Tests", () => {
     it.each(getPillTagsData)("getPillTags checking for:: %s", (testName, items, output) => {
         Utils.getPillTags(items).forEach((item, index) => { expect(item).toStrictEqual(output[index]); });
     });
+});
+
+describe("Utils isDeepEquals Tests", () => {
+    const isDeepEquals = [

```

Comment:
please add null/undefined also
please add more than one uid value also. equal and not equal